scss/bitstyles/objects/button--mode/button--mode.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Objects/Button--mode" />
+
+# Mode Buttons
+
+A set of buttons that switch between modes, such as a calendar-view selection.
+
+<Canvas isColumn>
+  <Story name="Button--mode">
+    {`
+      <div class="o-button--mode-group">
```

Comment:
In this PR we rename `button--group` to `button--grouped`. Should this also be `button--mode-grouped`, to align them? 🤷

Reply:
Aha! Thanks for spotting that — I had already renamed that class to `.o-button--mode-container`, but didn’t update it here. This is a container for the mode buttons, not a button in itself. The naming maybe isn’t perfect 🤔 I’ll run with `-container` for now, other suggestions are welcome :)